scripts/nb-tester/requirements.txt

```diff
@@ -0,0 +1,6 @@
+nbconvert==7.11.0
+nbformat==5.9.2
+ipykernel==6.27.1
+qiskit[all]==0.45.1
+qiskit-aer==0.13.1
+qiskit-ibm-runtime==0.16.1
```

Comment:
Thanks for pinning everything! I think this is fine to allow patch updates, since these projects follow semantic versioning.

```suggestion
# We pin to exact versions for a more reproducible and
# stable build.
nbconvert~=7.11.0
nbformat~=5.9.2
ipykernel~=6.27.1
qiskit[all]~=0.45.1
qiskit-aer~=0.13.1
qiskit-ibm-runtime~=0.16.1
```